python/lsst/ts/electrometer/csc.py

```diff
@@ -135,10 +187,14 @@ async def do_performZeroCalib(self, data):
             substates=[DetailedState.NOTREADINGSTATE],
             action="performZeroCalib",
         )
-        await self.report_detailed_state(DetailedState.CONFIGURINGSTATE)
-        await self.controller.perform_zero_calibration()
-        await self.report_detailed_state(DetailedState.NOTREADINGSTATE)
-        self.log.info("Zero Calibration Completed")
+        try:
+            await self.report_detailed_state(DetailedState.CONFIGURINGSTATE)
+            await self.controller.perform_zero_calibration()
+            await self.report_detailed_state(DetailedState.NOTREADINGSTATE)
+            self.log.info("Zero Calibration Completed")
+        except Exception:
+            self.log.error("Something went wrong")
```

Comment:
Please make this more useful or get rid of the try/except. For instance:

```
except Exception as e:
    self.log.error("performZeroCalib failed: {e!r}")
```

Or if you want a traceback then use log.exception -- in which case there's no need to capture the error as `e`.